activitytype-cql/src/main/java/io/nosqlbench/activitytype/cql/statements/binders/SimpleStatementValuesBinder.java

```diff
@@ -1,19 +1,55 @@
 package io.nosqlbench.activitytype.cql.statements.binders;
 
+import com.datastax.driver.core.ConsistencyLevel;
 import com.datastax.driver.core.SimpleStatement;
 import com.datastax.driver.core.Statement;
 import io.nosqlbench.virtdata.core.bindings.ValuesArrayBinder;
 
 /**
  * This binder is not meant to be used with anything but DDL or statements
- * which should not be trying to parameterize values in general. If this changes,
- * support will be added for parameterized values here.
+ * which should not be trying to parameterize values in general.
+ * Parametrized values are still possible through parametrized constructor parameter.
+ * This binder should be avoided in favor of binders returning PreparedStatement
  */
 public class SimpleStatementValuesBinder
         implements ValuesArrayBinder<SimpleStatement, Statement> {
 
+    private final boolean parametrized;
+
+    public SimpleStatementValuesBinder(boolean parametrized){
+        this.parametrized = parametrized;
+    }
+
     @Override
     public Statement bindValues(SimpleStatement context, Object[] values) {
-        return new SimpleStatement(context.getQueryString(), values);
+        String query = context.getQueryString();
+        if(parametrized) {
+            String[] splits = query.split("\\?");
+            assert splits.length == values.length+1;
+            StringBuilder sb = new StringBuilder();
+            sb.append(splits[0]);
+            for(int i = 1; i < splits.length; i++) {
+                sb.append(values[i - 1]);
```

Comment:
For string fields, this may require quote injection for non-primitive numeric types, etc. Since the non-parameterized form assumed the same syntax as used in prepared statements, quotes are not safe to assume. We should make them work co-compatibly.